.github/workflows/backend/prod/generate-swagger.yaml

```
@@ -0,0 +1,65 @@
+# This workflow will integrate the main branch with the main environment
+name: Generate Swagger file 
+
+on:
+  push:
+    branches:
+      - main
+    path:
+      'server/**'
+
+jobs:
+  generate-swagger:
+    name: Generate Swagger
+    runs-on: ubuntu-latest
+  
+  steps:
+    - name: Checkout code # checkout the code
+      uses: actions/checkout@v2
+
+    # .NET Setup
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '8.0.x' # change the version to the latest version that used
+    
```

we're using .net 9.0

```suggestion
        dotnet-version: '9.0.x'
    
```